bap-vibes/src/minizinc.ml

```diff
@@ -240,8 +227,12 @@ let serialize_mzn_params
             | Some s -> Set.add s t1)) in
   let temps = params.temps |> Var.Set.to_list in
   let temp_names =
-    List.map ~f:(fun t -> Var.sexp_of_t t |> Sexp.to_string) temps
+    List.map ~f:(fun t -> Var.to_string t) temps
   in
+  let hvars = List.map temps ~f:(fun temp -> Linear_ssa.orig_name (Var.to_string temp)) in
+  let hvars = String.Set.of_list (List.filter_opt hvars) in
+  let hvars = String.Set.diff hvars (String.Set.of_list (gpr @ regs)) in
```

Comment:
Hmm. Maybe I don't need to do this now that the convention is referring to "hvar_" prepended variables. I don't know that you'd want to force `hvar_SP` to be anything though for example